tutorials/sphinx-tutorials/multiagent_ppo.py

```diff
@@ -445,13 +445,13 @@
 
 policy = ProbabilisticActor(
     module=policy_module,
-    spec=env.unbatched_action_spec,
+    spec=env.single_action_spec,
```

Comment:
Also here the components are expecting the full one.

They might work without a full one but there might be subtle bugs I think.

Reply:
Nope I don't think so
1. it's a tutorial so if it passes it passes (it's not a public API that is likely to change)
2. `ProbabilisticActor` will look at your spec and do the required wrapping if it's not a Composite spec. And if the action is composite, so is `unbatched_action_spec`. So in either case it will work.

Reply:
It is that "will do the  required wrapping if it's not a Composite spec" that worries me. I don't think it knows the group name. I am afraid that this could lead to subtle bugs but I trust you

Reply:
if the full actions spec is not trivially "action" it returns the composite, so you're safe